src/proxy/inbound.rs

```diff
@@ -500,15 +500,15 @@ mod tests {
         sync::{Arc, RwLock},
     };
 
+    use crate::state::service::EndpointSet;
     use crate::{
         rbac::Connection,
         state::{
             self,
-            service::{endpoint_uid, Endpoint, Service},
+            service::{Endpoint, Service},
```

Comment:
**nit** Add EndpointSet here rather than on its own

Reply:
`cargo format` not being opionated is tragic